components/price/Table.vue

```diff
@@ -433,7 +439,7 @@
         {
           id: 7,
           title: "Custom Plugins",
-          openSourceText: 'Priority in plugin roadmap',
+          openSourceText: false,
```

Comment:
```suggestion
          openSourceText: true,
```
this is not true, OSS users can create custom plugins, no limits here